core/storage/improvements/gae_models.py

```diff
@@ -80,40 +86,34 @@ class TaskEntryModel(base_models.BaseModel):
     # The type of sub-entity a task entry focuses on. Value is None when an
     # entity does not have any meaningful sub-entities to target.
     target_type = ndb.StringProperty(
-        required=True, indexed=True, choices=TARGET_TYPES)
+        required=True, indexed=True, choices=TASK_TARGET_TYPES)
     # Uniquely identifies the sub-entity a task entry focuses on. Value is None
     # when an entity does not have any meaningful sub-entities to target.
     target_id = ndb.StringProperty(
         required=True, indexed=True)
 
-    # Utility field that gives a speedup of ~20% compared to querying by each of
-    # the invididual fields.
-    # Value has the form: "[entity_type].[entity_id].[entity_version]".
-    composite_entity_id = ndb.StringProperty(
-        required=True, indexed=True)
-
+    # A sentence generated by Oppia to describe why the task was created.
+    issue_description = ndb.StringProperty(
+        default=None, required=False, indexed=False)
     # Tracks the state/progress of a task entry.
     status = ndb.StringProperty(
-        required=True, indexed=True, choices=STATUS_CHOICES)
+        required=True, indexed=True, choices=TASK_STATUS_CHOICES)
     # ID of the user who closed the task, if any.
-    closed_by = ndb.StringProperty(
+    resolver_id = ndb.StringProperty(
```

Comment:
Can there be models that have `closed_by` set on prod or test server?

Reply:
And since you changed more models the questions is moreover if there are any models existing either on test or prod server.

Reply:
Nope, there aren't any models in production or test server yet b/c there's no functionality to do so yet (this PR introduces it, and even then it won't be used for a while longer).